BE/src/main/java/kr/codesquad/todolist/repository/JdbcCardRepository.java

```diff
@@ -31,9 +32,9 @@ public Card save(Card card) {
 
         KeyHolder keyHolder = new GeneratedKeyHolder();
         namedParameterJdbcTemplate.update(sql, new BeanPropertySqlParameterSource(card), keyHolder);
-        card.setId(Objects.requireNonNull(keyHolder.getKey()).longValue());
+        Long id = Objects.requireNonNull(keyHolder.getKey()).longValue();
 
-        return card;
+        return findById(id).orElseThrow(NoSuchElementException::new);
```

Comment:
다시 조회해오는 방식을 사용하셨군요~
정답은 없지만 새로운 Card 객체를 생성해서 넘겨주는 방식을 사용할 수도 있겠습니다 ^^

```suggestion
        return new Card(id, card);
```

Reply:
넵 현재는 다시 조회해오는 방식을 사용했는데, 
객체를 복사하는 방식은 고려하지 못했던것 같습니다. 감사합니다.